queries/t32/highlights.scm

```diff
@@ -97,14 +97,14 @@
   (identifier) @constant)
 (
  (argument_list (identifier) @constant.builtin)
- (#match? @constant.builtin "^[%/][a-zA-Z][a-zA-Z0-9.]*$")
+ (#lua-match? @constant.builtin "^[%%/][%l%u][%l%u%d.]*$")
 )
 
 (
   (command_expression
     command: (identifier) @keyword
     arguments: (argument_list . (identifier) @label))
-  (#match? @keyword "^[gG][oO][tT][oO]$")
+  (#lua-match? @keyword "[gG][oO][tT][oO]")
```

Comment:
It will also match `foogotobar` without `^ $`.

Reply:
whoops